airflow/hooks/S3_hook.py

```diff
@@ -69,6 +69,23 @@ def get_bucket(self, bucket_name):
         s3 = self.get_resource_type('s3')
         return s3.Bucket(bucket_name)
 
+    def create_bucket(self, bucket_name):
```

Comment:
We don't have a test for this method - can you add one please?

Reply:
Oops!

Tests added. Nice catch!